src/vcpkg/sourceparagraph.cpp

```diff
@@ -150,7 +165,7 @@ namespace vcpkg
         static constexpr StringLiteral DESCRIPTION = "Description";
         static constexpr StringLiteral FEATURE = "Feature";
         static constexpr StringLiteral MAINTAINERS = "Maintainer";
-        static constexpr StringLiteral NAME = "Source";
```

Comment:
This took me an embarrassing amount of time to realize that NAME was not always "name".